salt/config/schemas/esxvm.py

```diff
@@ -161,8 +168,10 @@ class ESXVirtualMachineScsiSchemaItem(ComplexSchemaItem):
 
     adapter = StringItem(
         title="Virtual Machine SCSI Controller Name",
-        description="Unique SCSI controller name"
-        "for creation an arbitrary value should be specified",
+        description=(
+            "Unique SCSI controller name"
+            "for creation an arbitrary value should be specified"
```

Comment:
Need a space here on the second line too?

Reply:
Fixed